process/sandbox-annual-review.md

```diff
@@ -0,0 +1,43 @@
+# Sandbox annual review 
+
+Sandbox projects are subject to an annual review by the TOC. This is intended to be a lightweight process to ensure that projects are on track, and getting the support they need.
+
+## How to file your annual review
+
+CNCF staff will notify the project maintainers when the project review is due. 
+
+Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [cncf/toc/reviews](https://github.com/cncf/toc/tree/master/reviews). 
+
+* Raise a PR titled *[Project name] Annual Review*
+* The PR should include a file called `annual-<project name>-<year>.md` (for example, `annual-amazingproj-2019.md`) with the contents described below
+* Send an email to the TOC mailing list so that the community knows the PR is there and can comment on it
```

Comment:
I would suggest to add a Link to the TOC mailing list here

Reply:
Yes, good idea. Would be even better if you used the suggest +- box to supply the link, to make it easy to act on :-)